lib/ansible/modules/network/cloudengine/ce_rollback.py

```diff
@@ -0,0 +1,523 @@
+                self.module.fail_json(
+                    msg='Error: Number of configuration checkpoints is not in the range from 1 to 80.')
+
+        # oldest check
+        if self.oldest:
+            if not self.oldest.isdigit():
+                self.module.fail_json(
+                    msg='Error: Number of configuration checkpoints is not digit.')
+            if int(self.oldest) <= 0 or int(self.oldest) > 80:
+                self.module.fail_json(
+                    msg='Error: Number of configuration checkpoints is not in the range from 1 to 80.')
+
+    def get_proposed(self):
+        """get proposed info"""
+
+        if self.commit_id:
+            self.proposed["commit_id"] = self.commit_id
+        if self.label:
+            self.proposed["label"] = self.label
+        if self.filename:
+            self.proposed["filename"] = self.filename
+        if self.last:
+            self.proposed["last"] = self.last
+        if self.oldest:
+            self.proposed["oldest"] = self.oldest
+
+    def get_existing(self):
+        """get existing info"""
+
+        if not self.rollback_info:
+            return
+        self.existing["RollBackInfos"] = self.rollback_info["RollBackInfos"]
+
+    def get_end_state(self):
+        """get end state info"""
+
+        self.end_state = None
+
+    def work(self):
+        """worker"""
+
+        self.check_params()
+        self.get_proposed()
+        # action mode
+        if self.action == "rollback":
+            if self.commit_id:
+                self.rollback_commit_id()
+            if self.label:
+                self.rollback_label()
+            if self.filename:
+                self.rollback_filename()
+            if self.last:
+                self.rollback_last()
+        elif self.action == "set":
+            if self.commit_id and self.label:
+                self.set_commitid_label()
+        elif self.action == "clear":
+            if self.commit_id:
+                self.clear_commitid_label()
+            if self.oldest:
+                self.clear_oldest()
+        elif self.action == "commit":
+            if self.label:
+                self.commit_label()
+        elif self.action == "display":
+            self.rollback_info = self.get_rollback_dict()
+
+        self.get_existing()
+        self.get_end_state()
+        self.results['changed'] = self.changed
+        self.results['proposed'] = self.proposed
+        self.results['existing'] = self.existing
+        self.results['end_state'] = self.end_state
+        if self.changed:
+            self.results['updates'] = self.updates_cmd
+        else:
+            self.results['updates'] = list()
+
+        self.module.exit_json(**self.results)
+
+
+def main():
+    """Module main"""
+
+    argument_spec = dict(
+        commit_id=dict(required=False),
+        label=dict(required=False, type='str'),
+        filename=dict(required=False, type='str'),
+        last=dict(required=False, type='str'),
```

Comment:
For integers please:
use `type='int`
remove the `isdigit` check from `check_params`